client.go

```diff
@@ -375,6 +382,22 @@ func (c *ovndb) MonitorTables(jsonContext interface{}) (*libovsdb.TableUpdates,
 	return c.client.Monitor(c.db, jsonContext, requests)
 }
 
+// AddLatestSchemaTables adds new tables that needs to be monitored
+// in SB/NB DB schema and currently not in SBTablesOrder or NBTablesOrder
+func (c *ovndb) AddLatestDBSchemaTables() {
+
+	dbSchema, err := c.client.GetSchema(c.db)
+	if err != nil {
+		log.Printf("Failed to get %s db schema: (%v)", c.db, err)
+		return
+	}
+	if c.db == DBSB {
+		if _, ok := dbSchema.Tables[TableChassisPrivate]; ok {
```

Comment:
I think it is better to use  an array and then walk through the elements of the array, instead of directly checking against TableChassisPrivate.

As in,

var latest SBTables = []string{TableChassisPrivate}

then you can loop through these tables to check whether they are in the dbSchema.Tables.